src/features/components/templates/footer/disclaimer/index.tsx

```diff
@@ -37,24 +37,30 @@ const Disclaimer = () => {
                     <Localize translate_text={item.data.text} components={item.data.components} />
                 </Typography.Paragraph>
             ))}
-            <Container.Fixed className={risk_warning_container} bgcolor="tertiary" padding="10x">
-                {visible_risk_content.map((item, index) => {
-                    const not_last_item =
-                        index !== visible_risk_content.length - 1 ? 'small' : undefined
-                    return (
-                        <Typography.Paragraph
-                            key={item.id}
-                            size={'small'}
-                            mb={not_last_item ? '10x' : undefined}
-                        >
-                            <Localize
-                                translate_text={item.data.text}
-                                components={item.data.components}
-                            />
-                        </Typography.Paragraph>
-                    )
-                })}
-            </Container.Fixed>
+            {is_non_eu && (
+                <Container.Fixed
+                    className={risk_warning_container}
+                    bgcolor="tertiary"
+                    padding="10x"
+                >
+                    {visible_risk_content.map((item, index) => {
+                        const not_last_item =
+                            index !== visible_risk_content.length - 1 ? 'small' : undefined
+                        return (
+                            <Typography.Paragraph
+                                key={item.id}
+                                size={'small'}
+                                mb={not_last_item ? '10x' : undefined}
```

Comment:
too much logic and `no` logic here 😄 

```suggestion
                        const current_margin =
                            index == visible_risk_content.length - 1 ?  undefined : '10x'
                        return (
                            <Typography.Paragraph
                                key={item.id}
                                size={'small'}
                                mb={current_margin}
```

Reply:
this is not related to this change, please provide a separate PR for you suggestion, thanks :pray: